src/Dapr.Extensions.Configuration/DaprSecretStoreConfigurationExtensions.cs

```diff
@@ -0,0 +1,91 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using Dapr.Client;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.DaprSecretStore;
+
+namespace Microsoft.Extensions.Configuration
+{
+    /// <summary>
+    /// Extension methods for registering <see cref="DaprSecretStoreConfigurationProvider"/> with <see cref="IConfigurationBuilder"/>.
+    /// </summary>
+    public static class DaprSecretStoreConfigurationExtensions
+    {
+        /// <summary>
+        /// Adds an <see cref="IConfigurationProvider"/> that reads configuration values from the Dapr Secret Store.
+        /// </summary>
+        /// <param name="configurationBuilder">The <see cref="IConfigurationBuilder"/> to add to.</param>
+        /// <param name="store">Dapr secret store name.</param>
+        /// <param name="secrets">The secrets to retrieve.</param>
+        /// <returns>The <see cref="IConfigurationBuilder"/>.</returns>
+        public static IConfigurationBuilder AddDaprSecretStore(
+            this IConfigurationBuilder configurationBuilder,
+            string store,
+            IEnumerable<DaprSecretDescriptor> secrets)
```

Comment:
Call this "secretNames"?  "secrets" may make it sound like the actual secret value.  KeyVault calls the equivalent "secretName" in their apis.

Reply:
Scratch "secretName".  But what about "secretDescriptors"?

Reply:
Changed to secretDescriptors in 22ecb65 cause the type holds more than just the names.